openapi/generator.go

```diff
@@ -344,11 +346,16 @@ func isResponseCodeRange(code string) bool {
 
 // setOperationResponse adds a response to the operation that
 // return the type t with the given media type and status code.
-func (g *Generator) setOperationResponse(op *Operation, t reflect.Type, code, mt, desc string, headers []*ResponseHeader) error {
+func (g *Generator) setOperationResponse(op *Operation, t reflect.Type, code, mt, desc string, headers []*ResponseHeader, example interface{}, examples map[string]interface{}) error {
 	if _, ok := op.Responses[code]; ok {
 		// A response already exists for this code.
 		return fmt.Errorf("response with code %s already exists", code)
 	}
+	if example != nil && examples != nil {
+		// Cannot set both 'example' and 'examples' values
+		return fmt.Errorf("set either 'example' or 'examples', but not both")
```

Comment:
Can we use a shorter, yet descriptive error message, such as: `'example' and 'examples' are mutually exclusive` ?